js/graviex.js

```diff
@@ -0,0 +1,991 @@
+            request['limit'] = limit; // default: 500
+        }
+        const response = await this.privateGetTradesHistory (this.extend (request));
+        return this.parseTrades (response, market, since, limit);
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {
+            'market': this.marketId (symbol),
+            'side': side,
+            'volume': amount.toString (),
+            'ord_type': type,
+        };
+        if (price !== undefined) {
+            request['price'] = price.toString ();
+        }
+        const response = await this.privatePostOrders (this.extend (request));
+        // {
+        //   "id": 288336422,
+        //   "at": 1644217763,
+        //   "side": "sell",
+        //   "ord_type": "limit",
+        //   "price": "0.141",
+        //   "avg_price": "0.0",
+        //   "state": "wait",
+        //   "market": "ltcbtc",
+        //   "created_at": "2022-02-07T10:09:23+03:00",
+        //   "volume": "0.141",
+        //   "remaining_volume": "0.141",
+        //   "executed_volume": "0.0",
+        //   "trades_count": 0,
+        //   "strategy": null
+        // }
+        return this.parseOrder (response);
+    }
+
+    async cancelOrder (id, symbol = undefined, params = {}) {
+        if (id === undefined) {
+            throw new ArgumentsRequired (this.id + ' cancelOrder() requires a side argument');
+        }
+        await this.loadMarkets ();
+        const request = {
+            'id': id,
+        };
+        const response = await this.privatePostOrderDelete (this.extend (request, params));
+        // {
+        //   "id": 288336422,
+        //   "at": 1644217763,
+        //   "side": "sell",
+        //   "ord_type": "limit",
+        //   "price": "0.141",
+        //   "avg_price": "0.0",
+        //   "state": "wait",
+        //   "market": "ltcbtc",
+        //   "created_at": "2022-02-07T10:09:23+03:00",
+        //   "volume": "0.141",
+        //   "remaining_volume": "0.141",
+        //   "executed_volume": "0.0",
+        //   "trades_count": 0,
+        //   "trades": [],
+        //   "strategy": null
+        // }
+        const order = this.parseOrder (response);
+        const status = order['status'];
+        if (status === 'closed' || status === 'canceled') {
+            throw new OrderNotFound (this.id + ' ' + this.json (order));
+        }
+        return order;
+    }
+
+    async cancellAllOrders (side = undefined, symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {
+        };
+        if (side !== undefined) {
+            request['side'] = side;
+        }
+        const response = await this.privatePostOrdersClear (this.extend (request, params));
+        const order = this.parseOrder (response);
+        const status = order['status'];
+        if (status === 'closed' || status === 'canceled') {
```

Comment:
let's also not throw an error here

Reply:
Okay. Fixed in the latest commit https://github.com/ccxt/ccxt/pull/11697/commits/162b106bdb42084917d6187a3a60fb6e0f55e7d1